presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1239,6 +1240,84 @@ public void testInsertForDefaultColumn()
 
     protected abstract TestTable createTableWithDefaultColumns();
 
+    @Test(dataProvider = "testColumnNameDataProvider")
+    public void testColumnName(String columnName)
+    {
+        if (!requiresDelimiting(columnName)) {
+            testColumnName(columnName, false);
+        }
+        testColumnName(columnName, true);
+    }
+
+    private void testColumnName(String columnName, boolean delimited)
+    {
+        String nameInSql = columnName;
+        if (delimited) {
+            nameInSql = "\"" + columnName.replace("\"", "\"\"") + "\"";
```

Comment:
please extract `qoute` method

Reply:
it's used once, and for specific purpose  and i prefer to leave it as is.
(i could as well make it part of data provided by the `@DataProvider`)